Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageLayoutOptionsResolver.swift

```diff
@@ -18,16 +18,27 @@ open class _ChatMessageLayoutOptionsResolver<ExtraData: ExtraDataTypes> {
         self.minTimeIntervalBetweenMessagesInGroup = minTimeIntervalBetweenMessagesInGroup
     }
 
+    @available(*, deprecated, message: "Use the same method with the appearance parameter instead")
+    open func optionsForMessage(
+        at indexPath: IndexPath,
+        in channel: _ChatChannel<ExtraData>,
+        with messages: AnyRandomAccessCollection<_ChatMessage<ExtraData>>
+    ) -> ChatMessageLayoutOptions {
+        optionsForMessage(at: indexPath, in: channel, with: messages, appearance: Appearance.default)
+    }
```

Comment:
we've got quite a few warnings in `ChatMessageLayoutOptionsResolver_Tests` file because of this change 😄 would be really cool if we can update the tests to access the updated new version of a func

Reply:
in `ChatMessageLayoutOptionsResolver_Tests.swift` (where the unit-tests for options resolver live) we keep calling the func which is now deprecated -> the compiler warns us about it just like it does for customers 🙂 The fix would be to invoke the new func with `Appearance` param added